File: Dragon_Dungeons.client/src/models/Player.js
import { RepoItem } from "./RepoItem.js"

export class Player extends RepoItem {
  constructor(data) {
    super(data)
    this.name = data.name
    this.picture = data.picture
    this.level = data.level
    this.class = data.class
    this.race = data.race
    this.characterId = data.characterId
    this.campaignId = data.campaignId
  }
}

File: Dragon_Dungeons/Models/Player.cs
namespace Dragon_Dungeons.Models;

public class Player : RepoItem
{

  public string Name { get; set; }
  public string Picture { get; set; }
  public string CharacterId { get; set; }
  public string CampaignId { get; set; }
  public int? Level { get; set; }
  public string Class { get; set; }
  public string Race { get; set; }
}